### ISSUE TITLE
Image name of image being visualized

### ISSUE THREAD

**Issue by surabhisnath** [yhenon/pytorch-retinanet#72]
Is there any way I can know which image name of test set is being visualized?

**Comment by r2ufuk** [comment]
The `CSVDataset` class keeps the names of images. If you are using `CocoDataset`, you might be able to implement a similar logic.

**Comment by surabhisnath** [comment]
I modified the dataloader and was able to do it. Thanks!
